src/parser/jobs/blu/modules/MoonFlute.tsx

```diff
@@ -0,0 +1,179 @@
+import {t} from '@lingui/macro'
+import {Trans} from '@lingui/react'
+import {ActionLink} from 'components/ui/DbLink'
+import {Action} from 'data/ACTIONS'
+import {dependency} from 'parser/core/Injectable'
+import {BuffWindow, EvaluatedAction, TrackedAction, ExpectedActionsEvaluator, ExpectedGcdCountEvaluator} from 'parser/core/modules/ActionWindow'
+import {HistoryEntry} from 'parser/core/modules/ActionWindow/History'
+import {GlobalCooldown} from 'parser/core/modules/GlobalCooldown'
+import {SEVERITY} from 'parser/core/modules/Suggestions'
+import React from 'react'
+
+const SEVERITIES = {
+	MISSING_EXPECTED_USES: {
+		1: SEVERITY.MINOR,
+		2: SEVERITY.MEDIUM,
+		3: SEVERITY.MAJOR,
+	},
+
+	TOO_FEW_GCDS: {
+		8: SEVERITY.MAJOR,
+	},
+}
+
+const EXPECTED_GCD_COUNT = 5
+// Some notes on the expected GCD count.
+// Waxing Nocturne is 15 seconds, so that seems like
+// it should be 6 GCDs, but the first .5 seconds you are still
+// in GCD recast time, so it's more like 14.5 seconds.
+//
+// If you are running very high spell speed, it is actually
+// possible to get those 6 GCDs, but that's not the entire story!
+//
+// If you just cats Moon Flute and then do 6 GCDs, then no matter
+// how strong those GCDs are -- even if they are 6 Revenge Blasts --
+// it will be a potency loss compared to 6 Revenge Blasts + 6 Sonic Booms.
+//
+// So while GCDs are as usual important, the big deal in a Moon Flute
+// window is weaving in as many oGCDs as we can *without* dropping a
+// GCD cast.
+//
+// And so our standard moon flute window actually only has us cast 4
+// GCDs (Triple Trident, The Rose of Destruction, Bristle, Matra Magic)
+// while giving up ~5 seconds to purposely clipping with oGCDs;
+// 3.6 seconds of Surpanakha, and the rest is all hard clips and
+// purposeful multi-weaves.
+//
+// ...but for implementation details, Phantom Flurry is marked as a GCD,
+// so even though we technically only do 4 GCDs in the window, we are
+// looking for a pseudo-5th, Phantom Flurry.
+
+export class MoonFlute extends BuffWindow {
+	static override handle = 'moonflutes'
+	static override title = t('blu.moonflutes.title')`Moon Flute Windows`
+
+	@dependency globalCooldown!: GlobalCooldown
+
+	override buffStatus = this.data.statuses.WAXING_NOCTURNE
+
+	override initialise() {
+		super.initialise()
+
+		const suggestionIcon = this.data.actions.MOON_FLUTE.icon
+		const suggestionWindowName = <ActionLink action="MOON_FLUTE" showIcon={false}/>
+		this.addEvaluator(new ExpectedGcdCountEvaluator({
+			expectedGcds: EXPECTED_GCD_COUNT, // 4 GCDs + Phantom Flurry _or_ 5 GCDs
+			globalCooldown: this.globalCooldown,
+			suggestionIcon,
+			suggestionContent: <Trans id="blu.moonflutes.suggestions.gcds.content">
+				Regardless of spell speed, ideally a <ActionLink action="MOON_FLUTE" /> window should contain at least
```

Comment:
This (and other suggestions) are *very* heavy on links with icons in them, which can get pretty noisy to look at. You might consider adding a showIcon={false} property to some of these to cut down on the visual clutter (especially if you have the same action or status linked multiple times in the same suggestion).

Reply:
I've sprinkled some showIcon false all over the place in 7625051a77f82f8daa110e5c408209fc9e88619c

Mentioned this on the discord, but the one hill I'm dying on is the ColdFog suggestions. Had several cases of folk who knew to proc Cold Fog, but didn't realize it gave you a new buff that allowed you to cast a different GCD, so I want the icons big and under the spotlight.